ideascube/views.py

```diff
@@ -265,8 +267,6 @@ def get(self, *args, **kwargs):
 ajax_proxy = AjaxProxy.as_view()
 
 
-def javascript(request):
-    response = render(request, 'ideascube/includes/js.html',
-                      {'settings': settings})
-    response['Content-Type'] = 'text/javascript'
-    return response
+@cache_control(max_age=31536000)
```

Comment:
You should be able to use the decorator in the urls.py directly, something like:

    url(r'^jsi18n/$', cache_control(max_age=31536000)(javascript_catalog)),